Shared/Samples/Show line of sight between geoelements/ShowLineOfSightBetweenGeoelementsView.swift

```diff
@@ -0,0 +1,358 @@
+                        isPresented = false
+                    }
+                }
+            }
+        }
+    }
+}
+
+private extension ShowLineOfSightBetweenGeoelementsView {
+    /// View model responsible for setting up the 3D scene, graphics, animation,
+    /// and performing line of sight analysis between moving and static geoelements.
+    @MainActor
+    @Observable
+    final class Model {
+        /// Set of predefined waypoints for animating the taxi's movement.
+        private let points = [
+            Point(x: -73.984513, y: 40.748469, spatialReference: .wgs84),
+            Point(x: -73.985068, y: 40.747786, spatialReference: .wgs84),
+            Point(x: -73.983452, y: 40.747091, spatialReference: .wgs84),
+            Point(x: -73.982961, y: 40.747762, spatialReference: .wgs84)
+        ]
+        
+        /// Height of the observer in meters. Updates the observer graphic when changed.
+        var height = 20.0 {
+            didSet {
+                changeObserverHeight(height)
+            }
+        }
+        
+        private var frameIndex = 0
+        private let frameMax = 120
+        private var pointIndex = 0
+        
+        var error: Error?
+        
+        /// The 3D scene containing basemap, elevation, and building layers.
+        let scene: ArcGIS.Scene = {
+            // Creates a scene and set an initial viewpoint.
+            let scene = Scene(basemapStyle: .arcGISImagery)
+            // Add base surface from elevation service.
+            let elevationSource = ArcGISTiledElevationSource(url: .elevationService)
+            let surface = Surface()
+            surface.addElevationSource(elevationSource)
+            scene.baseSurface = surface
+            var buildingLayer = ArcGISSceneLayer(url: .buildingsService)
+            scene.addOperationalLayer(buildingLayer)
+            return scene
+        }()
+        
+        /// Graphics overlay used to render the observer and target symbols.
+        var graphicsOverlay = GraphicsOverlay() {
+            didSet {
+                let renderer = SimpleRenderer()
+                renderer.sceneProperties.headingExpression = ("[HEADING]")
+                graphicsOverlay.renderer = renderer
+            }
+        }
+        
+        /// Overlay used to display the line of sight analysis visualization.
+        var analysisOverlay = AnalysisOverlay()
+        
+        private var lineOfSight: GeoElementLineOfSight?
+        private var taxiGraphic: Graphic?
+        private var displayLink: CADisplayLink!
+        private var observerGraphic: Graphic?
+        
+        var visibilityStatus = ""
+        
+        private let observerPoint = Point(
+            x: -73.984988,
+            y: 40.748131,
+            spatialReference: .wgs84
+        )
+        
+        private let taxiPoint = Point(
+            x: -73.984513,
+            y: 40.748469,
+            spatialReference: .wgs84
+        )
```

Comment:
```suggestion
        private let observerPoint = Point(
            latitude: 40.748131,
            longitude: -73.984988
        )
        
        private let taxiPoint = Point(
            latitude: 40.748469,
            longitude: -73.984513
        )
```
lat-lon instead of x-y. Also please add docstrings.